src/profile/script.js

```diff
@@ -88,9 +96,10 @@ function createCard({ oldData, newData, userId, username, profileDiffId }) {
   //looping through the new data to display in list
   for (const listItem in newData) {
     const li = document.createElement('li');
```

Comment:
Logic of  creating the DOM elements should be moved to util functions and reused them wherever is required